book/enterprise-contract.md

```diff
@@ -132,7 +132,6 @@ Additional Resources
 --------------------
 
 - [Enterprise Contract Documentation](https://enterprise-contract.github.io/)
-- [HACBS Documentation](https://red-hat-hybrid-application-cloud-build-services-documentation.pages.redhat.com/hacbs-documentation/)
```

Comment:
Is there a replacement for this URL?

Reply:
@arewm I did a quick look but didn't find anything.

Reply:
Found this Archived repo https://gitlab.cee.redhat.com/red-hat-trusted-application-pipeline/stonesoup-documentation

Reply:
There is https://redhat-appstudio.github.io/docs.appstudio.io ... but I don't know if that fits the same content. I don't know what used to be at the previous link.

Reply:
best to ask the EC team?

@lcarva @simonbaird ?

Reply:
I suppose this should link to the RHTAP Documentation. But isn't that what this repo is? Maybe it's redundant to have it listed on this page.